account_financial_report_qweb/wizard/general_ledger_wizard.py

```diff
@@ -25,7 +25,8 @@ class GeneralLedgerReportWizard(models.TransientModel):
     )
     date_from = fields.Date(required=True)
     date_to = fields.Date(required=True)
-    fy_start_date = fields.Date(required=True)
+    fy_start_date = fields.Date(
+        compute='compute_fy_start_date', readonly=True)
```

Comment:
You can remove readonly

Reply:
You are too perfectionist Pedro...

Reply:
Hehe, was only a comment FYI. Merging and I will squash as your commit is the correct one for giving you credits.